### PR TITLE
add Windows support

### PR DESCRIPTION
**Pull request by gillins**
This PR adds Windows support by adapting the provided makefile.vc and nmake.opt. I had to create a patch for freexl.c as it provides internal versions of `round()` and `lround()` when using VC, but the various VC versions are inconsistent with providing these (VC2015 provides both, VC2008 provides neither) so the easiest way seemed to rename the internal ones so they won't conflict with the VC ones where they exist.

I also had to add libiconv as a dependency since this package does use it - not sure how it was building on Linux etc before.

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+nmake /f makefile.vc
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipe/bld.bat]: Is there a `ctest` or something similar that we could run on Windows?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,11 +8,26 @@ source:
...
+    build:
+        - python  # [win]
+        - libiconv
+        
+    run:
+        - libiconv
+        
 test:
     commands:
         - test -f ${PREFIX}/lib/libfreexl.a  # [unix]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: If there is no testing at build time we could add something here to check if the lib was built.

File: recipe/bld.bat (hunk 1 above)
Comment (gillins) [inline recipe/bld.bat]: `ctest` is part of `cmake` but this isn't a cmake build and there is no `test` command in Windows unless you are running inside a .bat script. There is also no `make check` type target for freexl. I've adapted `run_test.py` from the kealib feedstock to check the libraries load and left the check for the static lib on Unix.

Comment (ocefpaf) [inline recipe/bld.bat]: Thanks for the explanation. (I thought I was starting to understand how to build stuff on Windows :smiley_cat:)

**Comment by ocefpaf** [comment]
LGTM

**Pull request merged**